### DISCUSSION TITLE
Mark individual notifications as read

### DISCUSSION THREAD

**Discussion by jenlith** [community/community#55197] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Going through my notifications, I took several minutes to figure out how to mark an individual notification as read. There's nothing to interact with that explicitly says "Mark as read." If that's what "Done" is, please rename it. "Done" makes me think it does something more, like approving a pull request. Other platforms use "Mark as read" so GitHub using the same phrase would make it much easier to understand.

**Comment by cehfisher** [comment]
Hi @jenlith - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!
